dvc/command/version.py

```diff
@@ -129,7 +127,12 @@ def get_supported_remotes():
             if not tree_cls.get_missing_deps():
                 supported_remotes.append(tree_cls.scheme)
 
-        return ", ".join(supported_remotes)
+        if supported_remotes.__len__() == 9:
+            return "All remotes"
+        elif supported_remotes.__len__() == 1:
```

Comment:
In Python, it's not good to use dunder methods directly(`__len__`). You can just use `len()`.